src/environments/environment.ts

```diff
@@ -3,7 +3,24 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+	production: false,
+	port: 'https://dev.sunbirdsaas.com/',
+	instance: '',
+	//instance: "https://sunbirdsaas.com/",
+	apiUrl: 'https://dev.sunbirdsaas.com/api',
```

Comment:
no urls to be added

Reply:
get rid of URLs and tokens from PR, and invalidate all tokens used here